Sushi-Go/main.py

```diff
@@ -1,3 +1,11 @@
-from tabletop import SushiGo
+import copy
```

Comment:
Try to keep main as a place to run other programs. We shouldn't have to import any libraries, only files